src/DepthChartSecondary.java

```diff
@@ -200,40 +196,71 @@ void printOffensivePersonnel(int personnel) {
             System.out.println(cb);
             cbArray[i] = cb;
             i++;
-        }
-        for (int j = cbArray.length - 1; j >= 0; j--) {
-            this.addPlayer(cbArray[j], "cb");
-        }
-    }else
+        }for(
+    int j = cbArray.length - 1;j>=0;j--)
```

Comment:
Keep an eye on code quality. I assume this will resolve itself when branches are merged.